### PR TITLE
Updating the Teams how to doc 

### PR DESCRIPTION
**Pull request by hachandr**

### CODE REVIEW

File: concepts/teams-create-group-and-team.md [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
 author: "nkramer"
 localization_priority: Priority
 ms.prod: "microsoft-teams"
 ---
 
-# Creating a group with a Microsoft Teams team
+# Creating teams and managing members using Microsoft Graph
 
-Creating a [group](/graph/api/resources/group?view=graph-rest-beta) that includes a [team](/graph/api/resources/team?view=graph-rest-beta) involves two steps: 
+While there are multiple ways to create teams for Microsoft Teams via Graph API, we recommend the following approach for best results.
```
Comment (Lauragra) [inline concepts/teams-create-group-and-team.md:11]: Revise as: "You can use the Microsoft Teams API in Microsoft Graph to create teams in multiple ways. This article describes the approach that we recommend for the best results."

File: concepts/teams-create-group-and-team.md [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
 
-- [Create a group](/graph/api/group-post-groups?view=graph-rest-beta) with the right properties.
-- [Add a team](/graph/api/team-put-teams?view=graph-rest-beta) to the group.
 
-## Create a group
+## Initial team creation
+
+All teams are backed by Office 365 Groups. When creating new teams via graph, setting up a new Office 365 Group, adding all owners and members and then converting that into a team is the quickest way to get your team up and running.
```
Comment (Lauragra) [inline concepts/teams-create-group-and-team.md:16]: The G in Office 365 Groups should be lowercase. Change "via graph" to "via Microsoft Graph". For readability, revise the second sentence as: "The quickest way to get your team up and running when you create new teams via Microsoft Graph is to set up a new Office 365 group, all all owners and members, and convert that into a team."

File: concepts/teams-create-group-and-team.md [hunk 3]
```diff
@@ -1,19 +1,21 @@
...
+
+1. Create an [Office 365 group](https://support.office.com/article/learn-about-office-365-groups-b565caa1-5c40-40ef-9915-60fdb2d97fa2) using the [Graph create group endpoint](https://docs.microsoft.com/graph/api/group-post-groups?view=graph-rest-beta&amp;tabs=http). If you are trying to set up a class team, please use the [Graph create EducationClass endpoint](https://docs.microsoft.com/graph/api/educationroot-post-classes?view=graph-rest-beta&amp;tabs=http). You may specify owners and members at this step. Ensure you have the right owners for the newly created team as described in Step 2.
```
Comment (Lauragra) [inline concepts/teams-create-group-and-team.md:18]: FYI, we always refer to Graph as "Microsoft Graph", per branding guidelines. For consistency we should refer to the links as methods or operations, not endpoints.

Comment (hachandr) [reply concepts/teams-create-group-and-team.md]: My apologies, I'll keep that in mind in the future!

**Comment by Lauragra** [comment]
@nkramer , if you approve this PR, we can get it merged in - thanks!

**Comment by hachandr** [comment]
@nkramer I've made the changes as per our conversation today! Thank you for the feedback.

**Pull request merged**